website/docs/d/msk_bootstrap_brokers.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+subcategory: "Managed Streaming for Kafka"
+layout: "aws"
+page_title: "AWS: aws_msk_bootstrap_brokers"
+description: |-
+    Get a list of brokers that a client application can use to bootstrap.
+---
+
+# Data Source: aws_msk_bootstrap_brokers
+
+Get a list of brokers that a client application can use to bootstrap.
```

Comment:
May probably make sense to add a note on what's the difference as opposite to `aws_msk_broker_nodes` data source? 🤔